news/2022-05-26-triangles.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: "triangles"
+date: 2022-05-26 23:00:00 +0000
+---
+
+Mappers and storyboarders unite in osu!'s newest contest: **triangles**!
+
+![](https://assets.ppy.sh/contests/147/header.jpg)
+
+[**lazer**](https://www.youtube.com/watch?v=ut_SOreClcc) is the next major update to osu!. If you [try it for yourself](https://github.com/ppy/osu#running-osu), you'll notice that osu!'s classic theme song has been replaced by *[cYsmix](https://osu.ppy.sh/beatmaps/artists/2) - triangles*:
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/2/Songs/cYsmix%20-%20triangles.mp3" type="audio/mpeg">
+</audio>
+
+**triangles** is a mapping and storyboarding contest using lazer's theme song, and the winning submission will be **bundled with all new installations of lazer** (along with other cool prizes!).
+
+There's a lot to explain. Let's dive in.
+
+## Teams
+
+To participate in this contest, you need a **team of 2-5 mappers/storyboarders** to create:
+
+- A **triangle-themed** beatmap spread...
+  - of the song *[cYsmix](https://osu.ppy.sh/beatmaps/artists/2) - triangles*
+  - following the [ranking criteria](/wiki/Ranking_Criteria) 
+  - with **FIVE DIFFICULTIES TOTAL**: Easy, Normal, Hard, Insane, and Expert
+- A **triangle-themed** storyboard...
+  - with a `.osb` file no greater than **1 MB**. This is to ensure the storyboard is well optimised and doesn't take too long to parse and load into the game.
```

Comment:
the .osb file size limit would be good to mention here, since it's important information. i'm sure storyboarders don't think of the storyboard as just the .osb file

Reply:
Isn't it already mentioned on this line? Confused.

Reply:
mapset size limit*

wow dunno how that happened